tools/tikv-control.md

```diff
@@ -98,15 +98,19 @@ In this command, the key is also the escaped form of raw key.
 
 To print the value of a key, use the `print` command.
 
-### Compact data manually
+### Compact data of each TiKV manually
 
-Use the `compact` command to manually compact TiKV data. If you specify the `--from` and `--to` options, then their flags are also in the form of escaped raw key. You can use the `--db` option to specify the RocksDB that you need to compact. The optional values are `kv` and `raft`.
+Use the `compact` command to manually compact data of each TiKV. If you specify the `--from` and `--to` options, then their flags are also in the form of escaped raw key. You can use the `--db` option to specify the RocksDB that you need to compact. The optional values are `kv` and `raft`. Also, the `--threads` option allows you to specify the concurrency that you compact and its default value is 8. Generally speaking, higher concurrency comes with faster compact speed, in spite of the fact that it makes an impact on the service. Therefore, it is recommended to choose an appropriate concurrency based on the circumstances.
```

Comment:
Modify the last two sentences to:

Generally, a higher concurrency comes with a higher compact speed, which might affect the service. You need to choose an appropriate concurrency based on the scenario.